department-of-veterans-affairs/va.gov-cms
# VACMS-4731: Add Vet Center listings view.

## Description
See #4731

## Testing done
VIsual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/111787172-04627c80-8895-11eb-9363-217562003de8.png)
![image](https://user-images.githubusercontent.com/2404547/111787186-075d6d00-8895-11eb-964f-82fdd953fe6e.png)

## QA steps
@kevwalsh - I added a block just below main content, and disabled (you'll need to enable it for qa). I tested as Vanessa, both with the Escanaba section added, and without, to ensure the edit buttons appear only when user has appropriate perms. You should be able to see the block on Escanaba VC, but nowhere else.
I used the smallest thumbnail size we have (I didn't see a 3:2 that is small enough to work in the grid), and operating status field is in the view, but said field isn't on any of the Escanaba nodes.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
